### ISSUE TITLE
[Bug]: Uploading a zip in the file explorer unzips it

### ISSUE THREAD

**Issue by amanape** [All-Hands-AI/OpenHands#4279]
### Is there an existing issue for the same bug?

- [X] I have checked the troubleshooting document at https://docs.all-hands.dev/modules/usage/troubleshooting
- [X] I have checked the existing issues.

### Describe the bug

When uploading a `.zip` type in the file explorer mid-session, the server unpacks it.

### Current OpenHands version

```bash
0.9.8
```

### Installation and Configuration

```bash
N/A
```

### Reproduction Steps

1. Start a new session
2. Drag a zip OR press the upload button and upload a `.zip` type
3. Observe the updated file explorer has the unpacked files

**Comment by amanape** [comment]
If we are uploading a project from the home screen, yes. This issue is specifically for uploading a zip mid-session through the file explorer upload functionality. I think the main issue here is that it will be unexpected to the user, or they may want to work on compressed files without unpacking them.

CC: @rbren

**Comment by rbren** [comment]
Yeah we probably only want to unzip when doing the project upload. Guessing this is a long-tail issue though, not high priority

**Comment by mamoodi** [comment]
Going to close this now because we removed upload functionality and all of this is handled in VS Code. So whatever functionality they support, OpenHands supports
